README.md

```diff
@@ -4,6 +4,30 @@
 
 Python package for searching for housing on Craigslist.
 
+-   Author: Ela Bandari, Junting He, Ling (Elina) Lin, Alex Truong
+
+
+## Overview
+
+Hunting for rental can be an exhausting and frustrating experience in Canada, but this process can be made easy with a simple installation of our package. This Python package intends to facilitate house hunting by scraping the listing information from Craigslist and organizing the extracted data for the users. Instead of having to manually go on the website to catch up with individual new listings, the users will be updated through email with new listings as per their selection criteria. 
```

Comment:
Hi Elina, 

Just some minor suggestions for changes:

1) Hunting for **rentals**....
2)  This Python package intends to facilitate **the house hunting process** by scraping the listing information from Craigslist and organizing the extracted data for **the user**. 
3) Instead of manually searching the website for new listings, **the user** will be updated on new listings meeting their selection criteria via email.